### ISSUE TITLE
Investigate whether an option should be added to NNCF config to control the effective learning rate of the compressoin parameters

### ISSUE THREAD

**Issue by vshampor** [openvinotoolkit/nncf#430]
In PyTorch, this may be implemented by setting up gradient hooks for trainable compression parameter tensors.

**Comment by vshampor** [comment]
Done for PyTorch, TF implementation coming in #790
